src/locales/tr.json

```diff
@@ -101,7 +101,7 @@
   },
   "games": {
     "quizGame": {
-      "title": "sınav oyunu"
+      "title": "Sınav oyunu"
```

Comment:
Can you also capitalize "o" and like "Sınav Oyunu"?

Reply:
ptal